.env

```diff
@@ -22,6 +22,12 @@ APP_SECRET=dc53e642f241baa0d6e8302d644c216e
 # MAILER_DSN=smtp://localhost
 ###< symfony/mailer ###
 
+###> API variables ###
+BASE_URI=https://od-api.oxforddictionaries.com
+APP_ID=7dd8530b
+APP_KEY=40b33363d5654b7e6786149f0f14f645
+###< API variables ###
+
```

Comment:
please remove this file and add it to .girignore file 
it should only be on your local